TorchSharp/TypeGeneration.cs

```diff
@@ -3581,7 +3637,7 @@ public long GetTensorDimension (int dim)
             get {
                     var dims = new long [Dimensions];
                     for (int i = 0; i < dims.Length; i++)
-                            dims [i] = (long)GetTensorDimension (i);
```

Comment:
What changed that allowed us to drop the cast?

Reply:
Nothing, in fact the cast is superfluous.